CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Removed a false warning about configuring the `GITLAB_TOKEN` environment variable when it's not needed.
 * Removed duplicate identifiers for XSIAM integrations.
 
+* Fixed an error message in checkbox validation.
```

Comment:
Elaborate. Something like "Fixed the error message in checkbox validation where the defaultvalue is wrong and added the name of the variable that should be fixed".

Reply:
Ok